README.md

```diff
@@ -21,4 +21,50 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
 
 ### Quick Start
 
-Go [here](docs/README.md) for an example of how to get up and going with the cluster api using vSphere.
\ No newline at end of file
+Go [here](docs/README.md) for an example of how to get up and going with the cluster api using vSphere.
+
+### Where to get the vSphere provider container
+
+The containers for this provider are currently hosted at `gcr.io/cnx-cluster-api/`.  Each release of the
+container are tagged with the release version appropriately.  Please note, the release tagging changed to
+stay uniform with the main cluster api repo.
+
+| release version | container url |
+| --- | --- |
+| 0.1.0 | gcr.io/cnx-cluster-api/vsphere-cluster-api-provider:v0.1 |
+| 0.2.0 | gcr.io/cnx-cluster-api/vsphere-cluster-api-provider:0.2.0 |
+
+To use the appropriate version (instead of `:latest`), replace the version in the generated `provider-components.yaml`,
+described in the quick start guide.
+
+### Compatibility Matrix
+
+Below are tables showing the compatibility between versions of the vSphere provider, the main cluster api,
+kubernetes versions, and OSes.  Please note, this table only shows version 0.2 of the vSphere provider.  Due
+to the way this provider bootstrap nodes (e.g. using Ubuntu package manager to pull some components), there
+were changes in some packages that broke version 0.1 (but may get resolved at some point) so the compatibility
+tables for that provider version are not provided here.
+                              
+Compatibility matrix for Cluster API versions and the vSphere provider versions.
+
+| | Cluster API 0.1.0 |
```

Comment:
will be great to add the link to the container `gcr.io/k8s-cluster-api/cluster-api-controller:0.1.0`